pinot-plugins/pinot-file-system/pinot-adls/src/main/java/org/apache/pinot/plugin/filesystem/ADLSGen2PinotFS.java

```diff
@@ -191,20 +182,22 @@ public void init(PinotConfiguration config) {
         clientSecretCredentialBuilder.httpClient(builder.build());
 
         dataLakeServiceClientBuilder.credential(clientSecretCredentialBuilder.build());
-        blobServiceClientBuilder.credential(clientSecretCredentialBuilder.build());
+        break;
+      }
+      case NONE: {
```

Comment:
`ANONYMOUS_ACCESS` may be a better naming for authentication type?

Reply:
done